README.rst

```diff
@@ -9,30 +9,195 @@ modelmeta
+2. Or just point ``pip`` to our `GitHub repo <https://github.com/pacificclimate/modelmeta>`_::
 
     $ pip install git+https://github.com/pacificclimate/modelmeta
 
-------------------------------
-What is Climate Coverage Data?
-------------------------------
+What is climate coverage data?
+==============================
+
+Climate coverage data (or "raster data" or "spatiotemporal data") consist of large data fields, typically over
+two or three dimensions in space plus a time dimension. Depending on the resolution in each axis, the data can
+typically be quite large in size. Typically there are several-to-many output quantities (e.g. temperature,
+precipiation, wind speed/direction) and often there can be multiple scenarios, multiple model implementations,
+and multiple runs of each model further exacerbating the size of the data.
+
+Managing database migrations
+============================
+
+Introduction
+------------
+
+Modifications to ``modelmeta``'s schema definition are now managed using
+`Alembic`_, a database migration tool based on SQLAlchemy.
+
+In short, Alembic supports and disciplines two processes of database schema change:
+
+- Creation of database migration scripts (Python programs) that modify the schema of a database.
+
+- Application of migrations to specific database instances.
+
+  - In particular, Alembic can be used to *create* a new instance of a ``modelmeta`` database by migrating an
+    empty database to the current state. This is described in detail below.
+
+For more information, see the `Alembic tutorial <http://alembic.zzzcomputing.com/en/latest/tutorial.html>`_.
+
+History
+-------
+
+The existing instance of a ``modelmeta`` database (``monsoon/pcic_meta``) was created prior to the adoption of
+Alembic, and therefore the timeline for Alembic database migrations is slightly confusing.
+
+Timeline:
+
+- *the distant past*: ``pcic_meta`` is created by mysterious primeval processes.
+
+- *somewhat later*: ``modelmeta`` is defined using SQLAlchemy, mapping most (but not all) features of the existing
+  ``pcic_meta`` database into an ORM.
+
+- 2017-07-18:
+
+    - Alembic is introduced.
+    - Alembic is used to create migration ``614911daf883`` that adds item ``seasonal`` to ``timescale`` Enum.
+
+- 2017-08-01:
+
+    - The SQLAlchemy ORM is updated to reflect all features of the ``pcic_meta`` database.
+      This mainly involves adding some missing indexes and constraints.
+
+    - Alembic is used to create a logically-previous migration ``7847aa3c1b39`` that creates the initial
+      database schema from an empty database.
+
+    - The add-seasonal migration is modified to logically follow the initial-create migration.
+
+Creating a new ``modelmeta`` database
+-------------------------------------
+
+There are in fact two tools in this package to create a new ``modelmeta`` database.
+
+- (DEPRECATED) Use the script ``scripts/mkblankdb.py``.
```

Comment:
I'm good with that. I didn't want to delete it until I checked with you that it didn't retain some use. I'll take that as a 'no'.